ios/RNInstabug/InstabugAPMBridge.m

```diff
@@ -36,33 +36,55 @@ - (id) init
     return self;
 }
 
+// This method is exporting a method
```

Comment:
I think this might be a little bit overkill since the literal method name is "RCT_EXPORT_METHOD" what do you think 🤔 ?